cnf-certification-test/operator/suite.go

```diff
@@ -63,6 +63,38 @@ func LoadChecks() {
 			testOperatorOlmSubscription(c, &env)
 			return nil
 		}))
+
+	checksGroup.Add(checksdb.NewCheck(identifiers.GetTestIDAndLabels(identifiers.TestOperatorHasSemanticVersioningIdentifier)).
+		WithSkipCheckFn(testhelper.GetNoOperatorsSkipFn(&env)).
+		WithCheckFn(func(c *checksdb.Check) error {
+			testOperatorSemanticVersioning(c, &env)
+			return nil
+		}))
+}
+
+// This function checks for semantic versioning of all installed operators
+func testOperatorSemanticVersioning(check *checksdb.Check, env *provider.TestEnvironment) {
+	check.LogInfo("Starting testOperatorSemanticVersioning")
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	allOperators := env.AllOperators
+	for _, operator := range allOperators {
+		operatorVersion := operator.Version
+		check.LogInfo("Testing Operator %q for version %s", operator, operatorVersion)
+
+		if provider.IsValidSemanticVersion(operatorVersion) {
+			check.LogInfo("Operator %q has semantic version %s", operator, operatorVersion)
+			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(operator.Namespace, operator.Name,
+				"Operator has semantic version ", true).AddField(testhelper.Version, operatorVersion))
+		} else {
+			check.LogError("Operator %q has invalid semantic versioning %s", operator, operatorVersion)
+			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewOperatorReportObject(operator.Namespace, operator.Name,
+				"Operator has semantic version ", false).AddField(testhelper.Version, operatorVersion))
```

Comment:
Shouldn't it be "Operator has an **invalid** semantic version"? as the reason for this operator to be non-compliant?